themes/default/content/docs/languages-sdks/python/python-blocking-async.md

```diff
@@ -0,0 +1,199 @@
+---
+title_tag: "Blocking and Async Python with Pulumi | Languages & SDKs"
+meta_desc: An overview of how to use Python with Pulumi for infrastructure as code on any cloud (AWS, Azure, Google Cloud, Kubernetes, etc.).
+title: Blocking and Async
+h1: Blocking and Async Python with Pulumi
+meta_image: /images/docs/meta-images/docs-meta.png
+menu:
+  languages:
+    parent: python
+    weight: 5
+aliases:
+- /python/async/
+---
+
+In large and complex Python Pulumi programs, blocking calls can prevent progress
+from being made by the Pulumi runtime by preventing the event loop from
+processing asynchronous tasks.  
```

Comment:
Should we start this with a brief description of how Pulumi Python programs rely on an event loop to handle resource declarations, outputs and applies, and more asynchronously? Then this sentence follows naturally.

Suggest nixing "In large and complex ..." and "may", I think we can be prescriptive: blocking calls block the Pulumi runtime.

Reply:
Yeah, these are good points. I mentioned large and complex, because in general it is a nothing burger for a small program, but when the stack is super complicated its much harder to tell where the slowdown is coming from.

I'll reword it

Reply:
Done